### ISSUE TITLE
I made a remix

### ISSUE THREAD

**Issue by coff33ninja** [beyarkay/eskom-calendar-api#4]
In case it looks bad. I present the repo, I build a desktop app with the API https://github.com/coff33ninja/Loadshedding-App
I am still learning Python and this seemed like a fun 4hour project to play with.

**Comment by beyarkay** [comment]
Looks cool! Thanks for letting me know, hearing about other devs using the API is really motivating.

Was there anything about the API you'd like to see change/improve? I'm always looking for (constructive) feedback (:

**Comment by coff33ninja** [comment]
If possible a method to use friendlier names per region/area. My plan with the gui is to manually create a json list to reference the API making the naming friendlier on output. Unless the API data output has an already built function I might have missed.

**Comment by beyarkay** [comment]
No, you're right we don't have something like that just yet. But it is in the works! Hopefully in the coming weeks
